code-challenges.js

```diff
@@ -4,36 +4,90 @@
 
 // Please read all questions thoroughly!
 // If you get stuck, please leave comments to help us understand your thought process.
-
 // --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
 
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 42
-// var temp = 350
-// var temp = 212
+var temp = [42, 350, 212]
+for (i = 0; i < temp.length; i++) {
+  if (temp[i] === 212){
+    console.log(`${temp[i]} is at boiling point`);
+  } else if (temp[i] < 212) {
+    console.log(`${temp[i]} is below boiling point`);
+  } else if (temp[i] > 212){
+    console.log(`${temp[i]} is above boiling point`);
+  }
+}
+
+
+// var temp = 350   var temp = 212
+
```

Comment:
I like the approach but you didn't use the test variable as provided. These code challenges are often a exercise in working from a given input.